app/modules/individuals/models.py

```diff
@@ -56,6 +58,37 @@ def remove_encounter(self, encounter):
     def get_members(self):
         return [encounter.owner for encounter in self.encounters]
 
+    def get_featured_asset_guid(self):
+        rt_val = None
+        if self.featured_asset_guid is not None:
+            if self.validate_presence_in_asset(self.featured_asset_guid):
+                rt_val = self.featured_asset_guid
+        elif len(self.encounters) > 0 and self.encounters[0].annotations is not None:
+            from app.modules.encounters.models import Encounter
+
+            encounter = Encounter.query.get(self.encounters[0].guid)
+
+            if len(encounter.annotations) > 0:
+                assert encounter.annotations[0].asset_guid
+                rt_val = self.encounters[0].annotations[0].asset_guid
+        return rt_val
+
+    def set_featured_asset_guid(self, asset_guid):
+        if self.validate_presence_in_asset(asset_guid):
+            self.featured_asset_guid = asset_guid
+
+    def validate_presence_in_asset(self, asset_guid):
```

Comment:
Surely this is validating that the asset_guid passed is present in this individual. 
The name imp[lies it's looking to see if an asset guid is in an asset...